content/training/Walk-through/pop_and_lab_data.Rmd

```diff
@@ -69,17 +69,21 @@ reporting_week <- "2017-W16"  # Set the reporting week
 The population estimates for regions and age groups are collected in order to calculate attack and mortality rates. **The template allows several ways to provide population estimates, but importantly, the data must be stored under specific variable names.** In your real-use case, the specific age groupings and region names in your data may differ from the examples, but the variable names must follow the expected standard.  
 
 You can do one of the following:  
-1) Enter the estimates directly **[need clarification here as this is not yet built]**  
+1) Import a spreadsheet file with the estimates in the correct format  
 2) Use the function gen_population() to derive the estimates from proportions  
-3) Import a spreadsheet file with the estimates in the correct format  
 
 ![](/images/AJS_Population_Counts.png)  
 
-For the AJS example, we can use an Excel spreadsheet containing population figures by region. Uncomment this line of code in the read_population_data chunk and modify it to import the Excel spreadsheet (link for spreadsheet download at top of this webpage). Note that the spreadsheet variables names have already been edited to match the expected names ("patient_origin" and "population").  
+**For this case study, we import an Excel spreadsheet containing population data by quartier (available for download in case study overview page).**  
 
-As they will not be used, delete or comment (#) all the code lines in the chunk that create fake data for population_data_region, population_data_age, and population_data_age_months.
+See the read_population_data chunk and modify the command to import the Excel spreadsheet. The command uses import() wrapped around here() to locate the file. The `which=` argument of import() specifies that the sheet "quartier" should be imported.  
+
+Note that the variables names in the spreadsheet have already been edited to match the expected variable names ("patient_origin" and "population").  
+
+*The import() function is wrapped around the here() function. The here() function makes it easy for R to locate files on your computer. It is best to save the dataset within your R project, and to provide here() with any R project subfolder names. You can read more about the here() function in the data import page of the R Basics tutorial.* 
```

Comment:
Putting something in italics doesn't really do much to make it stand out for users. Instead, it's better to use the notice shortcodes from Hugo: https://learn.netlify.com/en/shortcodes/notice/

Another important distinction is that `here()` will locate files *within a project*. If someone opens here and attempts to use it outside of a project, then they will not have good results.

```suggestion
{{% notice note %}}
The `import()` function is wrapped around the `here()` function. The `here()` function makes it easy for R to locate files within a project. It is best to save the dataset within your R project, and to provide `here()` with any R project subfolder names. You can read more about the `here()` function in the data import page of the R Basics tutorial.
{{% /notice %}}
```